kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -177,7 +177,7 @@ define([
                 if (typeof selectedItem === 'number') {
                     selectedItem = String(selectedItem);
                 }
-                return Validation.validateText(selectedItem, validationConstraints);
+                return Validation.validateTextString(selectedItem, validationConstraints);
```

Comment:
Another change was normalizing to `validateTextString` - there was a `validateText` alias, which seemed confusing and unnecessary. I went with `validateTextString` to complement `validateTextSet`.